specification/confidentialledger/resource-manager/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: confidentialledger
+  clear-output-folder: true
+```
+
```

Comment:
We need to add a `Go multiapi` section here, please take this file as an example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/alertsmanagement/resource-manager/readme.go.md